session-3-jshell/README.md

```diff
@@ -4,8 +4,33 @@ Java REPL (Read-Eval-Print-Loop): previously called Kulla.
 
 The JShell API can be used via applications as well.
 
-Look for the the JShell REPL in the `JAVA_HOME/bin` folder:
+Windows Users
 
+    Look for the the JShell REPL in the `JAVA_HOME\bin` folder:
+
+    c:\... dir /W %JAVA_HOME%\bin\js*
+    
+    Directory de C:\Program Files\Java\jdk-9\bin
+
+    jshell.exe     jsound.dll     jsoundds.dll   jstack.exe     jstat.exe
+    jstatd.exe
+
+    c:\... echo %JAVA_HOME%
+    "c:\Program Files\Java\jdk-9\"
+
+    c:\... echo %PATH%
+    c:\Program Files\Java\jdk-9\bin\;....
+     
+    Note: If JAVA_HOME is not setting on you PC (Or it's no a Java 9 version) or PATH doesn't show the bin folder for JDK-9, You should set the correct value following next steps: Invoke the Control Panel, and then System. Click Advanced and then Environment Variables. Add the location of the JDK-9 folder to the JAVA_HOME, and add or edit the location of the bin folder of the JDK installation to PATH variable in System Variables. You should set the JAVA_HOME value between quotation mark.
```

Comment:
Some language corrections and reformatting of text:

If `JAVA_HOME` is not set on you machine (or it's not set to Java 9) or `PATH` doesn't show the `bin` folder for JDK-9, You should set the correct value following next steps: 
- Invoke the Control Panel, and then System. 
- Click Advanced and then Environment Variables. 
- Add the location of the JDK-9 folder to the `JAVA_HOME`, 
- and add or edit the location of the bin folder of the JDK installation to `PATH` variable in System Variables. 

You should set the `JAVA_HOME` value between quotation mark.